src/_sass/commons/_variables-font-awesome.scss

```diff
@@ -249,7 +249,7 @@ $fa-var-exclamation-circle: "\f06a";
 $fa-var-exclamation-triangle: "\f071";
 $fa-var-expand: "\f065";
 $fa-var-expeditedssl: "\f23e";
-$fa-var-external-link: "\f08e";
```

Comment:
These were displaying on all of my browsers (including FF) as blank spaces, rather than missing codepoint glyphs, which may help explain why I didn't notice the issue, but anyway it's great to have them back!